docs/aks/README.md

```diff
@@ -492,9 +458,11 @@ Replace the values in the `AKS_Dashboard.json` file surrounded by `%%` with the
 ```bash
 
 cd $REPO_ROOT/docs/aks/demo
-sed -i "s/%%SUBSCRIPTION_GUID%%/${He_Sub}/g" AKS_Dashboard.json && \
+sed -i "s/%%SUBSCRIPTION_GUID%%/$(eval $He_Sub)/g" AKS_Dashboard.json && \
```

Comment:
[blocker] I ran into an error here with eval. He_Sub is the value of the subscription and not the command string. So the eval fails.